sources/packages/backend/apps/api/src/route-controllers/cas-supplier/models/cas-supplier.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { CASSupplierSiteStatus, SupplierStatus } from "@sims/sims-db";
+
+export class CASSupplierInfoAPIOutDTO {
+  dateCreated: Date;
+  supplierNumber?: string;
+  supplierProtected?: boolean;
+  supplierStatus: SupplierStatus;
+  isValid: boolean;
+  supplierSiteCode?: string;
+  addressLine1?: string;
+  siteStatus?: CASSupplierSiteStatus;
+  siteProtected?: string;
+}
+
+export interface AddCASSupplierAPIInDTO {
```

Comment:
API DTOs must be a class and have the validators as restricted as possible.
Please also add an E2E test to validate the bad request.